ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -342,14 +342,14 @@ public struct Avatar: View {
                           with: windowProvider)
     }
 
-    /// `AvatarCutout`: Cutout shape for an Avatar
+    /// `CircleCutout`: Cutout shape for a circle
     ///
     /// `xOrigin`: beginning location of cutout on the x axis
     ///
     /// `yOrigin`: beginning location of cutout on the y axis
     ///
-    /// `cutoutSize`: dimensions of cutout shape of the Avatar
-    public struct AvatarCutout: Shape {
+    /// `cutoutSize`: diameter of the circle to cut out
+    public struct CircleCutout: Shape {
```

Comment:
Does this struct need to be public?

Reply:
can it still be internal only but still in separate file outside of avatar.swift?

Reply:
+1 on keeping the struct internal and move it to a separate file (it may be useful for other controls beyond the Avatar/AvatarGroup).

Reply:
Moved it to a new file, but kept the file under Avatar. I considered moving it to Core or something, but seemed like a better idea to keep it in Avatar until we need it somewhere else?